### ISSUE TITLE
Using relayShow on Balloons while using Persistance is not showing remaining Balloons after app is being closed

### ISSUE THREAD

**Issue by KonradSzewczuk** [skydoves/Balloon#105]
**Please complete the following information:**
- Library Version [v1.2.3]
- Affected Device(s) [Pixel 2 API 27]
 
**Describe the Bug:**

I have a requirement to show the multiple tooltips only once during entire app lifetime (on the beginning)
I've chosen to use `relayShow` method (https://github.com/skydoves/Balloon#show-sequentially) for showing consecutive Balloons (I should show one, then after it is being dissmised i should show the second. Up to three Balloons)

However, I should also show every of that Balloon only once. So for example when I've shown the first Balloon and I closed my app, when I return I should be able to see the second Balloon and continue to the last Balloon from the `relayShow`.
I thought, that the https://github.com/skydoves/Balloon#persistence will be good for this scenario, but when I am showing the 
first Balloon and close the app, when I return I can not see the others.

Code: 

```
    fun ComponentActivity.baseTooltipBuilder(): Balloon.Builder {
     return Balloon.Builder(this)
        .setLayout(R.layout.v_tooltip_content)
        .setMarginLeft(16)
        .setMarginRight(16)
        .setCornerRadius(8f)
        .setArrowSize(12)
        .setArrowConstraints(ALIGN_ANCHOR)
        .setArrowColorResource(R.color.blue)
        .setBackgroundDrawableResource(R.drawable.tooltip_background)
        .setBalloonAnimation(FADE)
        .setElevation(0)
        .setFocusable(false)
        .setDismissWhenClicked(true)
        .setLifecycleOwner(this)
}
     
            val firstBalloon = activity.baseTooltipBuilder().setPreferenceName("test1").setShowTime(1).build()
            val secondBalloon = activity.baseTooltipBuilder().setPreferenceName("test2").setShowTime(1).build()
            val thirdBalloon = activity.baseTooltipBuilder().setPreferenceName("test3").setShowTime(1).build()

            firstBalloon.relayShow(secondBalloon, secondView).relayShow(thirdBalloon, thirdView)
            firstBalloon.show(firstView)
```

**Expected Behavior:**

When I am using `relayShow` with the https://github.com/skydoves/Balloon#persistence, when I trigger the `relayShow` from the first Balloon, see it and close the app, when I return to the app I should see the remaining Balloons from the `relayShow` 

@skydoves Is it possible to achieve this requirement?

**Comment by KonradSzewczuk** [comment]
@skydoves any ideas? I want to use `relayShow` for that, I could even use my own persistance mechanism, but is there a way to call some sort of listener on each balloon that that I can handle? For example if there is some listener for telling me if the balloon is showing I could just store the information that it was shown already in my own persistance mechanism during `relayShow`

**Comment by KonradSzewczuk** [comment]
@skydoves any ideas? I want to use `relayShow` for that, because it has some lifecycle handling implemented. I can handle the persistence logic myself, but I see that during `relayShow` the `setOnBalloonClickListener` or `setOnBalloonOutsideTouchListener` are not being invoked, don't they?

**Comment by skydoves** [comment]
The `relay` function is implemented in `setOnBalloonDismissListener` to the previous balloon.
So it doesn't have any list or stack which needs memory and the previous one can not know about what is next balloon. (They related to only events each other.) It seems to need persisting a list of balloons, and those statuses for resolving this issue.

**Comment by KonradSzewczuk** [comment]
@skydoves but can you confirm if those listeners should still be invoked during `relay`? Because they don't work for me now. I can then update those statuses about which Balloon is being displayed at the moment, but I can't do that because of that

**Comment by skydoves** [comment]
Hmm, could you please try again with `setFocusable(false)`?

**Comment by KonradSzewczuk** [comment]
@skydoves With `setFocusable(false)` it is being called but I want to avoid that. What about when I `setPreferenceName("..")` on each Balloon when set up the `relayShow`, is there possibility to still of them being saved somewhere? How can I found them and get those preferences eventually?

**Comment by skydoves** [comment]
Released (#206) a new stable `1.3.6`!
